File: JavaScript_Courses/JavaScript_Basics/Preparation/02_Conditional_Statements_Lab/07_Area_of_Figures.js
function areaOfFigures(input) {
    let figure = input[0];
    let area = 0;

    if (figure == "square") {
        let side = Number(input[1]);

        area = side * side;
    } else if (figure == "rectangle") {
        let sideA = Number(input[1]);
        let sideB = Number(input[2]);

        area = sideA * sideB;
    } else if (figure == "circle") {
        let radius = Number(input[1]);
        area = Math.PI * radius * radius;
    } else if (figure == "triangle") {
        let side = Number(input[1]);
        let h_side = Number(input[2]);
        area = side * h_side / 2;
    }

    console.log(`${area.toFixed(3)}`);
}

// areaOfFigures(["square", "5"]);
// areaOfFigures(["rectangle", "7", "2.5"]);
// areaOfFigures(["circle", "6"]);
// areaOfFigures(["triangle", "4.5", "20"]);

File: Python_Courses/Python_Basics/Preparation/02_Conditional_Statements_Lab/07_Area_of_Figures.py
# from 11.11.2020

import math
fig_type = input()

if fig_type == "square":
    a = float(input())
    area = a * a
    print(f'{area:.3f}')

elif fig_type == "rectangle":
    a = float(input())
    b = float(input())
    area = a * b
    print(f'{area:.3f}')
elif fig_type == "circle":
    a = float(input())
    area = math.pi * a * a
    print(f'{area:.3f}')
elif fig_type == "triangle":
    a = float(input())
    h = float(input())
    area = a * h / 2
    print(f'{area:.3f}')
